src/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixer.php

```diff
@@ -61,40 +61,53 @@ public function isCandidate(Tokens $tokens)
      */
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
-        $functionyTokens = $this->getFunctionyTokenKinds();
-        $languageConstructionTokens = $this->getLanguageConstructionTokenKinds();
-        $braceTypes = $this->getBraceAfterVariableKinds();
+        static $logicalTokenKinds = [
```

Comment:
Can we have tests for these cases?

Reply:
I have amended the commit to add "non-regression" tests 🙂 Thanks for reviewing